lib/regexutil/syntax/compile.go

```diff
@@ -0,0 +1,296 @@
+// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
Please add a comment here:
```go
// TODO: drop this file when the issue at <link_to_go_issue> is fixed
```

This comment will help us returning back to regexp/syntax package after upgrading Go builder to the Go version with the bugfix.

Please file an issue at Go repository, so it could be linked here.

Please add the same comment to every file in this package.